backend/data_distro/api_tests/schema/revision.jsonnet

```diff
@@ -0,0 +1,32 @@
+local Base = import 'APITestBase.libsonnet';
+local Types = Base.Types;
+
+// LOTS of optional fields here. 
```

Comment:
@eazapata23 this schema is probably the worst offender, but there are a _lot_ of optional fields throughout. If you can, sync with @jadudm or some folks working on the XLS side about these. My understanding is that the new submission flow will make lots of fields mandatory.

Reply:
Again, I'd be happy with a ticket that breaks every one of these out as a checklist where we have to come back and revisit the schema. How we reconcile new data (that might not be optional) vs. old data (which might have `null`s) might require a schema that says "if the year field is 2022, or 2021, or..., then this field *might* be `null`" whereas something checking 2023 might be different. 

That said, exactly how we chose to tackle that is not something I want to solve now. I *do* want us to have a ticket that explicitly lists out each schema, and forces us to revisit them and start answering these "harder" questions. So, @danswick is not wrong... we want to address this. However, I'd rather move this PR through and into the tree, and ticket it for a future sprint/increment.